internal/cli/atlas/deployments/search/indexes/describe.go

```diff
@@ -28,8 +29,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var describeTemplate = `ID	NAME	DATABASE	COLLECTION	STATUS
-{{.IndexID}}	{{.Name}}	{{.Database}}	{{.CollectionName}}	{{.Status}}
+var describeTemplate = `ID	NAME	DATABASE	COLLECTION	STATUS	TYPE
+{{.IndexID}}	{{.Name}}	{{.Database}}	{{.CollectionName}}	{{.Status}}	{{if .Type}}{{.Type}}{{else}}` + search.DefaultType + `{{end}}
+`
+
+var describeOutputTemplate = `ID	NAME	DATABASE	COLLECTION	STATUS	TYPE
```

Comment:
nit: maintaining two templates might not be ideal in the long run (non blocking)